platform/database_objects/config_cloud.py

```diff
@@ -0,0 +1,2 @@
+# string to configure the database connection - localhost (will have to change to cloud in the future)
+postgres_url = "postgresql://<user>:<password>@localhost:5432/kbb_db"
```

Comment:
We are going to use mysql, same as AWS RDS and image for dockerfile.db. Possible to change to mysql here?

Reply:
@cherrythia I'm working on the mysql here, encountered some issues atm. Please give me some time to get it resolved. As for the qn on the virtual env, typically the virtual env for local and production is the same, we just have to take care of the requirements.txt and make sure that both are using the same requirements.txt, as pip install takes care of it on the background (correct me if I'm wrong, coz that's my understanding).

Reply:
@cherrythia made the updates for mysql.